src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star-active.svg);
```

Comment:
Use inactive star here

Reply:
If this is done, then the asterisks will be located differently than in the task condition